test/issue-28.test.js

```diff
@@ -359,7 +359,7 @@ test('Nested wildcards with parametric and static - 2', t => {
   })
 
   findMyWay.lookup(
-    { method: 'GET', url: '/foo3/bar1' },
+    { method: 'GET', url: '/foo3/bar1', headers: {} },
```

Comment:
is this change `headers: {}`  needed in all those cases?

Reply:
Yes, because `lookup` will directly search in `req.headers['accept-version']`.

Reply:
Nope, `lookup` accepts node *request* and *response* objects, in the test we are using a basic object for simplicity.